templates/download/server/thank-you.html

```diff
@@ -13,11 +13,11 @@
 </noscript>
 {% endif %}
 
-<section class="p-strip is-bordered u-no-padding--top u-no-padding--bottom">
+<section class="p-strip u-no-padding--top u-no-padding--bottom" style="overflow: visible;">
   <div class="p-strip--suru-topped">
     <div class="row">
       <div class="col-8">
-        <h1 style="font-weight: 100;">Thank you for downloading Ubuntu Server</h1>
+        <h1>Thank you for downloading Ubuntu Server {{ version }}</h1>
```

Comment:
When I subscribe to Ubuntu Server weekly news, the new page says “Thank you for downloading Ubuntu Server” again. But I’m not downloading Ubuntu Server again, I’m downloading a PDF. Either the form submission should not go to a new page, or this heading should change to “Thanks for subscribing” or similar (which would make the notification redundant).